screens/FollowerList.tsx

```diff
@@ -0,0 +1,179 @@
+
+  const handleReject = () => {
+    console.log("Nutzer abgelehnt.");
+  };
+
+  const initialUsers: User[] = [
+    {
+      id: "1",
+      username: "top_G_3",
+      nickname: "Aleks069",
+      avatar: require("../assets/images/Max.jpeg"),
+      posts: [
+        require("../assets/images/Ei.jpeg"),
+        require("../assets/images/Luca.jpeg"),
+      ],
+      isFollowing: true,
+    },
+    {
+      id: "2",
+      username: "koenig_kemal",
+      nickname: "Kevin",
+      avatar: require("../assets/images/Kevin.jpeg"),
+      posts: [
+        require("../assets/images/Adrian.jpeg"),
+        require("../assets/images/Luca.jpeg"),
+        require("../assets/images/Aleks.jpeg"),
+      ],
+      isFollowing: true,
+    },
+  ];
+
+  const [users, setUsers] = useState<User[]>(initialUsers);
+
+  const handleFollowPress = (id: string) => {
+    setUsers((currentUsers) =>
+      currentUsers.map((user) =>
+        user.id === id ? { ...user, isFollowing: !user.isFollowing } : user,
+      ),
+    );
+  };
+
+  return (
+    <ScrollView>
+      <FlatList
+        style={{ marginVertical: 10 }}
+        data={users}
+        keyExtractor={(item) => item.id}
+        renderItem={({ item }) => (
+          <View style={styles.item}>
+            <TouchableOpacity
+              onPress={() => {
+                navigation.navigate("FollowerProfile", { user: item });
+              }}
+              style={styles.touchable}
+            >
+              <Image source={{ uri: item.avatar }} style={styles.avatar} />
+              <Text style={styles.name}>{item.username}</Text>
+            </TouchableOpacity>
+            {type === 2 ? (
+              <>
+                {" "}
+                <TouchableOpacity
+                  style={{ marginRight: "1%" }}
+                  onPress={() => handleAccept()}
+                >
+                  <Ionicons
+                    style={{ color: COLORS.green }}
+                    size={SIZES.large}
+                    name="checkmark-outline"
+                  ></Ionicons>
+                </TouchableOpacity>
+                <TouchableOpacity onPress={() => handleReject()}>
+                  <Ionicons
+                    style={{ color: COLORS.red }}
+                    size={SIZES.large}
+                    name="close-outline"
+                  ></Ionicons>
+                </TouchableOpacity>
+              </>
+            ) : (
+              <TouchableOpacity
+                style={styles.button}
+                onPress={() => handleFollowPress(item.id)}
+              >
+                {/* hier muss Prüfung rein, ob schon gefolgt oder nicht */}
+                <Text style={{ fontSize: SIZES.small }}>
+                  {type === 0 && "Folgen"}
+                  {type === 1 && "Entfolgen"}
+                </Text>
+              </TouchableOpacity>
+            )}
+          </View>
+        )}
+      />
+    </ScrollView>
+  );
+};
+
+const styles = StyleSheet.create({
```

Comment:
Kann man hier ein eigenes Stylesheet machen? Dann gehört das in Stylesheets ordner

Reply:
ja, würde das aber generell einmal einheitlich festlegen und einordnen zusammen. Auf einem eigenen Branch und einheitlich für alle stylesheets in den tsx files. Bis dahin würde ich die styles erstmal in den jeweiligen files lassen, sonst wirds schnell unübersichtlich